.github/workflows/nx-integration-tests.yml

```diff
@@ -0,0 +1,186 @@
+name: NX Integration Test
+#description: Workflow to run NX assisted integration tests for Mage-OS projects.

```

Comment:
Any reason this is commented?

Reply:
You can't put descriptions in workflows, only actions.
Felt the description was useful, as they are used in the actions.
Happy to remove though, what do you think?

Reply:
Haha, okay, interesting. All fine, no strict opinion on this.